ego/cli/sign.go

```diff
@@ -134,6 +137,9 @@ func (c *Cli) signExecutable(path string) error {
 // JSON could not be unmarshalled
 func (c *Cli) readConfigJSONtoStruct(path string) (*config.Config, error) {
 	data, err := c.fs.ReadFile(path)
+	if os.IsNotExist(err) {
+		return nil, ErrConfigDoesNotExist
```

Comment:
```suggestion
		return nil, fmt.Errorf("%w: %v", errConfigDoesNotExist, path)
```
A bit unorthodox, but maybe it's helpful for the user to get the path in the error message, especially in the enclave.json case. What do you think?

(I think you would usually define a custom error type for this, but I guess this is what you mean by "more advanced error handling in Go is a bit fugly", hence this proposal.)